mobile.css

```diff
@@ -0,0 +1,292 @@
+@import url(./style.css);
+
+@media screen and (min-width: 375px) {
+  #hero {
+    background-image: url(./images/main-bg-img.png);
+    background-size: cover;
+  }
+
+  .menu-bar {
+    margin-top: 10px;
+  }
+
+  .menu-bar i {
+    font-size: 40px;
+  }
+
+  .header-logo-container {
+    width: 100%;
+    margin: 2rem auto 0 auto;
+    display: flex;
+    align-content: center;
+  }
+
+  .header-logo {
+    width: 5rem;
+    height: 5rem;
+    border-radius: 50%;
+    animation: image-rotation 10s infinite linear;
+  }
+
+  #nav-wrapper {
+    position: fixed;
+    width: 100%;
+    min-height: 100%;
+    overflow: hidden;
+    top: -100%;
+    left: 0;
+    background-color: #fff;
+
+    /* display: none; */
+    transition: all 0.5s ease;
+  }
+
+  #nav-wrapper.open {
+    top: 0;
+  }
+
+  #nav-wrapper i {
+    transform: scale(1.5);
+    color: var(--theme-color);
+  }
+
+  #nav-wrapper i:hover {
+    transform: scale(1.5);
+    transition: all 700ms ease;
+  }
+
+  .nav-items {
+    width: 100%;
+    text-align: center;
+    padding-top: 3rem;
+  }
+
+  .nav-items li {
+    padding: 12px 16px;
+    font-size: 20px;
+    font-weight: 800;
+    text-decoration: none;
+    color: var(--theme-color);
+  }
+
+  .nav-items a {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  .nav-items li:hover {
+    background-color: var(--theme-color);
+    border-radius: 5px;
+    color: var(--gray-color);
+    font-style: italic;
+    transition: all 800ms;
+    transform: scale(1.2);
+  }
+
+  #nav-wrapper.active {
+    display: block;
+    opacity: 1;
+  }
+
+  #intro-section {
+    margin-top: 4rem;
+    padding-bottom: 3rem;
+  }
+
+  #intro-header h3 {
+    color: var(--theme-color);
+    font-size: 2rem;
+  }
+
+  #intro-header h1 {
+    font-size: 3rem;
+    background-image: url(./images/orange-wave-bg.jpg);
+    background-size: cover;
+    color: transparent;
+    background-clip: border-box; /* Standard property */
+    -webkit-background-clip: text; /* Vendor-prefixed version */
+    font-weight: 800;
+  }
+
+  .intro-text-container {
+    width: 95%;
+    margin: 3rem auto;
+    border: 3px solid #fff;
+  }
+
+  .intro-text-container p {
+    padding: 10px 15px;
+    font-size: 16px;
+  }
+
+  #date-time-container {
+    width: 95%;
+    margin: auto;
+  }
+
+  #date-time-container h4 {
+    color: #4e4949;
+    font-weight: 900;
+    font-size: 1.8rem;
+  }
+
+  #date-time-container p {
+    margin-top: 0.5rem;
+    color: #4e4949;
+    font-size: 1rem;
+  }
```

Comment:
- [ ] Kindly write DRY CSS code. Lines 128 & 135 can be written better without duplication by using a common class and assigning it the color color: #4e4949;. DRY code is easy to manitain.
 
- [ ] Please use classes to style common elements that share a common design. This reduces the code you need to write & improves maintanability